pkg/kubernetes/config/kubeconfig_test.go

```diff
@@ -121,9 +121,10 @@ func TestInvalidExistingConfig(t *testing.T) {
 	testDataDir := "./testdata/"
 	testDataPath := "./testdata/test_kubeconfig"
 	defer os.RemoveAll(testDataDir)
-	os.Mkdir(testDataDir, 0777)
+	err := os.Mkdir(testDataDir, 0777)
+	assert.NoError(t, err)
```

Comment:
I would make this `require.NoError()` because nothing is going to work properly without a directory to store things in.